src/Styleguide/Pages/Artwork/index.js

```diff
@@ -114,6 +118,16 @@ export class Artwork extends Component {
             </TabContent>
           </Col>
         </Row>
+        <Row>
+          <Col xl={8} lg={8} md={8} sm={12} xs={12}>
```

Comment:
Are these colspans at various breakpoints?

Reply:
Yep, that exactly.